src/DGtal/math/linalg/EigenDecomposition.ih

```diff
@@ -215,10 +215,8 @@ decomposeQL( Matrix& V, Vector& d, Vector e )
       // otherwise, iterate.
       if( m > l )
         {
-          int iter = 0;
```

Comment:
not related but clean looks fine ;)

Reply:
yeah I know.. my bad.. I saw this warning and couldn't resist to fix it ;)